streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/discovery/ambari/ComponentPropertyPattern.java

```diff
@@ -48,7 +48,8 @@
   // KAFKA
   // protocol (plaintext, ssl, kerberos, etc?), host (can be empty), port
   // https://cwiki.apache.org/confluence/display/KAFKA/Multiple+Listeners+for+Kafka+Brokers
-  KAFKA_BROKER("listeners", Pattern.compile("([a-zA-Z]+)://[a-zA-Z0-9_\\-\\\\.]*:([0-9]+)")),
+  // no matching pattern as listeners can contains multiple URIS with different schemes/protocols
+  KAFKA_BROKER("listeners", null),
```

Comment:
minor: Let's either commenting that it will be handled separately while importing cluster or just removing KAFKA_BROKER from here.

Reply:
I started with removing that field but it is used in other places like [here](https://github.com/hortonworks/streamline/blob/master/streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/register/impl/KafkaServiceRegistrar.java#L38
) and [here](https://github.com/hortonworks/streamline/blob/master/streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/service/metadata/KafkaMetadataService.java#L63). I did not want to remove that and comeup with a different enum or string from some other class as part of this PR.

Reply:
OK thanks for checking. Let's leave it as it is.